index.html

```diff
@@ -3798,60 +3795,59 @@ <h2>DID Resolution</h2>
 href="#metadata-structure">metadata structure</a>. This structure contains
 metadata about the <a>DID document</a> contained in the
-<code>did-document</code> or <code>did-document-stream</code>. This metadata
-typically does not change between invocations of the <code>resolve</code>
+<code>didDocument</code> property. This metadata
+typically does not change between invocations of the <code>resolveRepresentation</code>
 function unless the <a>DID document</a> changes, as it represents data about
 the <a>DID document</a>. If the resolution is unsuccessful, this output MUST
 be an empty <a href="#metadata-structure">metadata structure</a>. Properties
 defined by this specification are in <a
-href="#did-document-metadata-properties"></a>.
+href="#did-document-metadata"></a>.
             </dd>
         </dl>
 
         <p>
 Conforming <a>DID resolver</a> implementations do not alter the signature of
-these functions in any way. <a>DID resolver</a> implementations might map the
-<code>resolve</code> and <code>resolveRepresentation</code> functions to a
+these functions in any way. <a>DID resolver</a> implementations might map
+this function to a
 method-specific internal function to perform the actual <a>DID resolution</a>
 process. <a>DID resolver</a> implementations might implement and expose
 additional functions with different signatures in addition to the
-<code>resolve</code> function specified here.
+<code>resolveRepresentation</code> function specified here.
         </p>
 
         <section>
-            <h3>DID Resolution Input Metadata Properties</h3>
+            <h3>DID Resolution Options</h3>
 
             <p>
 The possible properties within this structure and their possible values are
 registered in the DID Specification Registries [[?DID-SPEC-REGISTRIES]]. This
-specification defines the following common properties.
+specification defines the following common properties for DID resolution
+options:
             </p>
 
             <dl>
                 <dt>
 accept
                 </dt>
                 <dd>
-The MIME type expressed as an <a data-lt="ascii string">ASCII string</a> of the
-caller's preferred <a>representation</a> of the <a>DID
-document</a>. The <a>DID resolver</a> implementation SHOULD use this value to
-determine the <a>representation</a> contained in the returned
-<code>did-document-stream</code> if such a <a>representation</a> is supported and
-available. This property is OPTIONAL. It is only used if the
-<code>resolveRepresentation</code> function is called and MUST be ignored if the
-<code>resolve</code> function is called.
+The MIME type of the caller's preferred <a>representation</a> of the <a>DID
+document</a>. The MIME type MUST be expressed as an  <a data-lt="ascii
+string">ASCII string</a>. The <a>DID resolver</a> implementation SHOULD use this
+value to determine the <a>representation</a> contained in the returned
+<code>didDocumentStream</code> if such a <a>representation</a> is supported and
```

Comment:
What is its not supported... what if I ask for:

```
accept: application/did+dag+cbor, application/did+json
``

Prefer to see a note encouraging or discouraging alignment with https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406

Reply:
There's a "representationNotSupported" error defined in the spec. Also note that DID Resolution has no dependency on HTTP.

Reply:
@OR13 does @peacekeeper's response answer your question?

Reply:
yes.